### PR TITLE
Bump to gtplib 3.2.0

### PR DESCRIPTION
**Pull request by javiermtorres**

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	{eradius, "2.2.4"},
 	{regine, "1.1.0"},
 	{cut, "1.0.3"},
-	{gtplib, "3.0.0"}
+	{gtplib, {git, "https://github.com/travelping/gtplib.git", {tag, "3.2.0"}}}
```
Comment (RoadRunnr) [inline rebar.config:10]: can't you use the hex release instead?

Comment (vkatsuba) [inline rebar.config:10]: 1 - the rebar lock is not updated
2 - why you don't use a hex.pm?

Comment (javiermtorres) [reply rebar.config:10]: I didn't know a hex package was preferred to a tag in source

Comment (vkatsuba) [reply rebar.config:10]: > There's no rebar.lock in this project

Yep, sorry my bad - we were remove it some time ago.

File: rebar.config [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	{eradius, "2.2.4"},
 	{regine, "1.1.0"},
 	{cut, "1.0.3"},
-	{gtplib, "3.0.0"}
+	{gtplib, "3.2.0"}
```
Comment (vkatsuba) [inline rebar.config:10]: `===> Package not found in any repo: gtplib 3.2.0`

Comment (vkatsuba) [reply rebar.config:10]: Fixed

**Review by vkatsuba — APPROVED** [review]

**Review by RoadRunnr — APPROVED** [review]
LGTM

File: rebar.config (hunk 2 above)
Comment (javiermtorres) [reply rebar.config:10]: Thanks!

**Pull request merged**